analysis/src/Utils.ml

```diff
@@ -71,3 +71,42 @@ let flattenLongIdent ?(jsx = false) ?(cutAtOffset = None) lid =
   in
   let path, _ = loop lid in
   List.rev path
+
+let identifyPexp pexp =
```

Comment:
Just a debug util I tend to use. Can remove this later, or move it elsewhere.

Reply:
This one is useful. Needed so many times.
Let's put it somewhere accessible.

Reply:
Yeah I think I wrote it by hand like 5 times first before I realized I must save it somewhere 😄